plugins/bgnotify/bgnotify.plugin.zsh

```diff
@@ -76,6 +76,7 @@ function bgnotify_appid {
 function bgnotify {
   local title="$1"
   local message="$2"
+  local icon="$3"
```

Comment:
How about something like this?

```diff
-    local icon="$3"
+    local icon="${3=$ZSH/plugins/bgnotify.png}"
```

Reply:
Bundling icons is maybe the neatest, yeah. I'd be interested to hear what the maintainers think about that approach.

Rather than changing the line you suggest, I'd argue that the icon should be put inside the function `bgnotify_formatted`.  And I'd argue that it is more helpful to have one icon for success and a different for failure. Attached is a screenshot of my own setup, but maybe everyone is not as fond of wolves as I am? :sweat_smile:

![swappy-20231124_201728](https://github.com/ohmyzsh/ohmyzsh/assets/1308179/d1260a63-073f-4d0f-a7f3-14470dc8c86b)

Reply:
> ![swappy-20231124_201728](https://private-user-images.githubusercontent.com/1308179/285532444-d1260a63-073f-4d0f-a7f3-14470dc8c86b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDA5OTU1MjQsIm5iZiI6MTcwMDk5NTIyNCwicGF0aCI6Ii8xMzA4MTc5LzI4NTUzMjQ0NC1kMTI2MGE2My0wNzNmLTRkMGYtYTdmMy0xNDQ3MGRjOGM4NmIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTEyNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzExMjZUMTA0MDI0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZWYxNTg5MGMwNWZiYjk4Yjc0NTgwMjAwOTE5OTAyMzg5Y2IzODUxM2U0N2M1NmYzOWM1Mjk0MzAyNzlkZmU2YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.1ClFzJdjyPiY2WVPU2RFqxf9BpR5dziX1QX2xQhuD2w)

:rofl: that's a neat one.

Yeah, it's a neat idea. I just found about [zsh-notify](https://github.com/marzocchi/zsh-notify) where it looks like this is already implemented

Reply:
Looks nice, thanks for the heads up. And I see that others also like wolves and dogs. 😎 🐺  Much amaze.
https://gist.github.com/marzocchi/14c47a49643389029a2026b4d4fec7ae